lib/cellular/models/sms.rb

```diff
@@ -21,12 +23,13 @@ def deliver
       @delivered = true
     end
 
-    def deliver_later
-      Cellular::Jobs::AsyncMessenger.perform_async options
+    def deliver_async(delivery_options = {})
+      Cellular::Jobs::AsyncJob.set(delivery_options).perform_later(options)
     end
 
     def deliver_at(timestamp)
-      Cellular::Jobs::AsyncMessenger.perform_at timestamp, options
+      warn "[DEPRECATION] 'deliver_at' is deprecated; use 'deliver_async' instead"
```

Comment:
Since this change is probably backwards incompatible, can't we just remove this method altogether while we're at it?